website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -35,12 +35,16 @@ When using defer, it compares artifacts from the most recent successful producti
 
 ### Defer in the dbt Cloud IDE
 
+For deferral in the IDE, you must have production artifacts available generated by a deploy job. dbt Cloud will first check for these artifacts in your Staging environment (if it exists) or, if not, the Production environment. 
+
+The defer feature in the IDE is not expected to work if a Staging environment exists but no deploy job has run there. The necessary metadata to power the defer feature will not exist until a deploy job has run successfully in the Staging environment.
+
 To enable defer in the dbt Cloud IDE, toggle the **Defer to production** button on the command bar. Once enabled, dbt Cloud will:
```

Comment:
to address some of my [initial feedback on UI inconsistency](https://github.com/dbt-labs/docs.getdbt.com/pull/6856#issuecomment-2630569382)
```suggestion
To enable defer in the dbt Cloud IDE, toggle the **Defer to staging/production** button on the command bar. Once enabled, dbt Cloud will:
```